ches-backend/chesService/controller.js

```diff
@@ -36,4 +36,19 @@ const emailMerge = async (req, res, next) => {
   }
 };
 
-module.exports = {sendEmail, emailMerge};
+
+const emailPreview = async (req, res, next) => {
+  const svc = getService();
+  let data = {};
+  try {
+    data = req.body;
+
+    let response = await svc.preview(data);
```

Comment:
This can be a `const` - the response object isn't getting reassigned anywhere else

Reply:
addressed in next commit.